scripts/lib/libvirt/templates/cpu-default.xml

```diff
@@ -0,0 +1,16 @@
+<cpu match='minimum'>
+    <model>qemu64</model>
+    <feature policy='require' name='fxsr_opt'/>
+    <feature policy='require' name='mmxext'/>
+    <feature policy='require' name='lahf_lm'/>
+    <feature policy='require' name='sse4a'/>
+    <feature policy='require' name='abm'/>
+    <feature policy='require' name='cr8legacy'/>
+    <feature policy='require' name='misalignsse'/>
+    <feature policy='require' name='popcnt'/>
+    <feature policy='require' name='pdpe1gb'/>
+    <feature policy='require' name='cx16'/>
+    <feature policy='require' name='3dnowprefetch'/>
+    <feature policy='require' name='cmp_legacy'/>
+    <feature policy='require' name='monitor'/>
+  </cpu>
```

Comment:
They are gone probably because libvirt is parsing the XML and outputting the result of the parsed data type. If this is not causing some error in the code, I would prefer to remove this space in the templates/ dir files, because some one will eventually scratch this itching